### PR TITLE
Update to Outdated files in the dev-1.18-ko.1 branch.

### PR DESCRIPTION
**Pull request by ysyukr**
### 34 of 37 files to be modified
- [x] content/ko/docs/concepts/architecture/nodes.md
- [x] content/ko/docs/concepts/cluster-administration/cluster-administration-overview.md
- [x] content/ko/docs/concepts/configuration/assign-pod-node.md
- [x] content/ko/docs/concepts/containers/container-lifecycle-hooks.md
- [x] content/ko/docs/concepts/containers/runtime-class.md
- [x] content/ko/docs/concepts/overview/working-with-objects/annotations.md
- [x] content/ko/docs/concepts/overview/working-with-objects/labels.md
- [x] content/ko/docs/concepts/overview/working-with-objects/names.md
- [x] content/ko/docs/concepts/services-networking/ingress.md
- [ ] content/ko/docs/concepts/services-networking/network-policies.md
> 원문의 문장 끝 공백 제거로 미반영
- [x] content/ko/docs/concepts/services-networking/service.md
- [x] content/ko/docs/concepts/storage/volume-pvc-datasource.md
- [x] content/ko/docs/concepts/storage/volumes.md
- [x] content/ko/docs/concepts/workloads/controllers/cron-jobs.md
- [x] content/ko/docs/concepts/workloads/controllers/deployment.md
- [x] content/ko/docs/concepts/workloads/pods/ephemeral-containers.md
- [ ] content/ko/docs/concepts/workloads/pods/init-containers.md
> #19603 에서 반영됨
- [x] content/ko/docs/concepts/workloads/pods/pod-topology-spread-constraints.md
- [x] content/ko/docs/home/_index.md
- [x] content/ko/docs/reference/_index.md
- [ ] content/ko/docs/reference/glossary/kube-scheduler.md
> #19712 진행시 문제가 있어 #19708 과 함께 진행됨
- [x] content/ko/docs/reference/glossary/replication-controller.md
- [x] content/ko/docs/setup/_index.md
- [x] content/ko/docs/setup/learning-environment/minikube.md
- [x] content/ko/docs/setup/production-environment/container-runtimes.md
- [x] content/ko/docs/setup/production-environment/windows/user-guide-windows-containers.md
- [x] content/ko/docs/tasks/administer-cluster/cluster-management.md
- [x] content/ko/docs/tasks/manage-kubernetes-objects/declarative-config.md
- [x] content/ko/docs/tasks/manage-kubernetes-objects/imperative-command.md
- [x] content/ko/docs/tasks/manage-kubernetes-objects/kustomization.md
- [x] content/ko/docs/tasks/run-application/horizontal-pod-autoscale.md
- [x] content/ko/docs/tasks/tools/install-minikube.md
- [x] content/ko/examples/application/deployment.yaml
- [x] content/ko/examples/application/simple_deployment.yaml
- [x] content/ko/examples/application/update_deployment.yaml
- [x] content/ko/examples/controllers/daemonset.yaml
- [x] content/ko/examples/controllers/nginx-deployment.yaml

### 1 files to be renamed
- [x] R098 content/ko/docs/concepts/containers/container-environment-variables.md -> content/ko/docs/concepts/containers/container-environment.md

### 1 files to be deleted
- [x] content/ko/docs/setup/production-environment/windows/user-guide-windows-nodes.md

from #19860 
/language ko

### CODE REVIEW

**Comment by ysyukr** [comment]
#19872 과 중복되는 항목에 대해 업데이트 진행했습니다.

File: content/ko/docs/concepts/services-networking/ingress.md [hunk 1]
```diff
@@ -115,6 +116,84 @@ spec:
...
+    {{< note >}}
+    경로의 마지막 요소가 요청 경로에 있는 마지막 요소의
+    하위 문자열인 경우에는 일치하지 않는다(예시: 
+    `/foo/bar` 와 `/foo/bar/baz` 와 일치하지만, `/foo/barbaz` 는 일치하지 않는다).
+    {{< /note >}}
+
+#### 다중 일치
+경우에 따라 인그레스의 여러 경로가 요청과 일치할 수 있다.
+이 경우 가장 긴 일치하는 경로가 우선하게 된다. 두 개의 경로가
+여전히 동일하게 일치하는 경우 접두사 경로 유형보다
```
Comment (gochist) [inline content/ko/docs/concepts/services-networking/ingress.md]:
```suggestion
여전히 동일하게 일치하는 경우 접두사(prefix) 경로 유형보다
```
앞서 기술된 Path Type을 연상할 수 있도록 병기를 제안드립니다. 참고 바랍니다.

File: content/ko/docs/concepts/services-networking/ingress.md [hunk 2]
```diff
@@ -115,6 +116,84 @@ spec:
...
+정확한 경로 유형을 가진 경로가 사용 된다.
```
Comment (gochist) [inline content/ko/docs/concepts/services-networking/ingress.md]:
```suggestion
정확한(exact) 경로 유형을 가진 경로가 사용 된다.
```

앞서 기술된 Path Type을 연상할 수 있도록 병기를 제안드립니다. 참고 바랍니다.

**Comment by ysyukr** [comment]
@gochist 리뷰 감사합니다.
확인해서 업데이트 및 커밋 스쿼시 진행했습니다.

**Comment by gochist** [comment]
@ysyukr 감사합니다!

/lgtm
/approve

**Pull request merged**